src/shared/pages/list/foiAccountManagers/accountManagersView.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { Reducer, useEffect, Fragment } from 'react';
+import { RouteComponentProps } from 'react-router';
+import { History } from 'history';
+import { getListItems } from '../../../services/entityListService';
+import { State } from './state';
+import { Action } from './actions';
+import { reducer } from './reducer';
+import { initialState } from './initialState';
+import {
+    GENERAL_ERROR_TITLE,
+    LOAD_FOI_ACCOUNT_MANAGERS_ERROR_DESCRIPTION
+} from '../../../models/constants';
+import ErrorSummary from '../../../common/components/errorSummary';
+import ErrorMessage from '../../../models/errorMessage';
+import useError from '../../../hooks/useError';
+import { Link } from 'react-router-dom';
+import EntityListItem from '../../../models/entityListItem';
+
+interface MatchParams {
+    teamId: string;
+}
+
+type TeamMembersProps = RouteComponentProps<MatchParams>;
+
+const onAddClick = (history: History) => {
+    history.push('/manage-foi-account-managers/add');
+};
+
+const FoiAccountManagersView: React.FC<TeamMembersProps> = ({ history, match }) => {
```

Comment:
Anyway to make this more generic? ideally passing in the lists? and FOI specific stuff?